meta-genivi-dev/meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "This project provides zebkit node.js bindings."
+SUMMARY = "zebkit"
+HOMEPAGE = "https://github.com/barmalei/zebra"
+# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
```

Comment:
This text is boiler plate that should not be here.  I think I mentioned it in the first review.  We need to know that the given license information is correct, at least as far as you can find out by studying the source code license information.